.gitignore

```diff
@@ -9,3 +9,4 @@
 .env
 .env_test
 certs
+config.yml
```

Comment:
No i think its not. It's in `.gitignore` cause we don't have to modify it anymore in github and if i modify it on local it will not be pushed here. If you clone the repo you have to create your own `config.yml` file also.